### PR TITLE
feat(onyx-1112): sell frame step

### PR DESCRIPTION
**Pull request by nickskalkin**
The type of this PR is: **Feat**

This PR solves [ONYX-1112]

### Description

Adds Frame step:

image

[ONYX-1112]: https://artsyproduct.atlassian.net/browse/ONYX-1112?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      mutation useUpdateMyCollectionArtworkMutation(
+        $input: MyCollectionUpdateArtworkInput!
+      ) {
+        myCollectionUpdateArtwork(input: $input) {
+          artworkOrError {
+            ... on MyCollectionArtworkMutationSuccess {
+              artwork {
+                title
+                internalID
+              }
```
Comment (dariakoko) [inline src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx:17]: What do you think about returning images here as well?

Comment (nickskalkin) [reply src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx:17]: Do we need images here?

**Review by dariakoko — APPROVED** [review]

File: src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx (hunk 1 above)
Comment (dariakoko) [reply src/Apps/Sell/Mutations/useUpdateMyCollectionArtwork.tsx:17]: I was wondering, if the user added new images, went to see the updated artwork in my collection and the images are still processing, what would we show?

File: src/Apps/Sell/Routes/__tests__/FrameRoute.jest.tsx [hunk 2]
```diff
@@ -35,10 +51,10 @@ beforeEach(() => {
       push: mockPush,
       replace: mockReplace,
     },
-    match: { location: { pathname: "/sell/submissions/submission-id/frame" } },
+    match: { location: { pathname: "/sell/submissions/externalId/frame" } },
```
Comment (dariakoko) [inline src/Apps/Sell/Routes/__tests__/FrameRoute.jest.tsx:54]: Is externalId here the same as submission-id? Let's use submission-id and stay consistent

**Pull request merged**